app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,12 @@
+
+
+<section id="projects">
+<% @contestants.each do |contestant| %>
+<h2>
+<%= contestant.name %> </h2>
+
+<p><%= contestant.project %></p>
```

Comment:
I would maybe consider naming your model method `project_list` or something a bit more descriptive. Just to not confuse yourself thinking contestants can only be on one project! But great work encapsulating this in the model method